### ISSUE TITLE
Error parsing go.mod. Unable to install gnostic and the plugin.

### ISSUE THREAD

**Issue by penthaapatel** [google/gnostic-grpc#68]
The `go.mod` file lists `github.com/googleapis/gnostic` as the current module’s module path, whereas it should be updated to `github.com/google/gnostic` instead it seems. While installing gnostic and the plugin, the following error appears and the installation is aborted.

**Steps to reproduce the error:**
On running the command
```
$  go get -u github.com/google/gnostic
```
the error produced is:
```
go get: github.com/google/gnostic@v0.3.0 updating to
	github.com/google/gnostic@v0.5.5: parsing go.mod:
	module declares its path as: github.com/googleapis/gnostic
	        but was required as: github.com/google/gnostic
```
On running the command
```
$  go get -u github.com/google/gnostic-grpc
```
The error produced is:
```
go get: github.com/google/gnostic-grpc@none updating to
	github.com/google/gnostic-grpc@v0.1.1: parsing go.mod:
	module declares its path as: github.com/googleapis/gnostic-grpc
	        but was required as: github.com/google/gnostic-grpc
```

**Comment by LorenzHW** [comment]
As a workaround, you can run:
`go get -u github.com/googleapis/gnostic`
and
`go get -u github.com/googleapis/gnostic-grpc`

**Comment by penthaapatel** [comment]
Thanks! That works.

**Comment by LorenzHW** [comment]
Fixed with #71
